pkg/action/upgrade_test.go

```diff
@@ -277,6 +277,59 @@ func TestUpgradeRelease_ReuseValues(t *testing.T) {
 	})
 }
 
+func TestUpgradeRelease_ResetThenReuseValues(t *testing.T) {
+	is := assert.New(t)
+
+	t.Run("reset then reuse values should work with values", func(t *testing.T) {
+		upAction := upgradeAction(t)
+
+		existingValues := map[string]interface{}{
+			"name":        "value",
+			"maxHeapSize": "128m",
+			"replicas":    2,
+		}
+		newValues := map[string]interface{}{
+			"name":        "newValue",
+			"maxHeapSize": "512m",
+			"cpu":         "12m",
+		}
+		newChartValues := map[string]interface{}{
+			"memory": "256m",
```

Comment:
Maybe I am missing something, but where does this come from?

Reply:
This is testing that values added to the chart between initial setup and upgrade are applied in the release values.

Reply:
Yeah I just could not find another occurrence of `256m` in this repository.

Reply:
Is it an issue ?
At least we know for sure that the value is not coming from elsewhere 😅

Reply:
Finally got it—this variable is used twice. Sorry for noise.